test/releasers/java-yoshi.ts

```diff
@@ -368,6 +368,141 @@ describe('JavaYoshi', () => {
     await releasePR.run();
     req.done();
   });
+  it('creates a snapshot PR, when latest release sha is head', async () => {
+    const versionsContent = readFileSync(
+      resolve(fixturesPath, 'released-versions.txt'),
+      'utf8'
+    );
+    const pomContents = readFileSync(resolve(fixturesPath, 'pom.xml'), 'utf8');
+    const graphql = JSON.parse(
+      readFileSync(resolve(fixturesPath, 'empty-commits.json'), 'utf8')
+    );
+    const req = nock('https://api.github.com')
```

Comment:
That mock chain is intense bro.

Reply:
Yes, this is just copied from the other snapshot PR that has commits. It might be better to start moving to sinon and mocking our github wrapper rather than the GitHub requests.